### PR TITLE
Label right and add euro currency

### PR DESCRIPTION
**Pull request by mateuszpmroz**

### CODE REVIEW

File: app/utils.js [hunk 1]
```diff
@@ -422,6 +427,27 @@ function refreshExchangeRates() {
...
+					global.exchangeRates['btceur'] = exchangeRate;
+					global.exchangeRatesUpdateTime = new Date();
+
+					debugLog("Using exchange rates: " + JSON.stringify(global.exchangeRates) + " starting at " + global.exchangeRatesUpdateTime);
+
+				} else {
+					debugLog("Unable to get exchange rate data");
+				}
+			} else {
+				logError("3825isdgij", {error:error, response:response, body:body});
```
Comment (KedziaPawel) [inline app/utils.js:447]: 3825isdgij what is this?

File: app/utils.js [hunk 2]
```diff
@@ -204,7 +204,12 @@ function formatExchangedCurrency(amount, exchangeType) {
 		if (exchangeType.toLowerCase() === 'usd' && global.exchangeRates['usdt'] != null && global.exchangeRates['usdtusd'] != null) {
 			dec = dec.times(global.exchangeRates['usdt']).times(global.exchangeRates['usdtusd']);
 			var exchangedAmt = parseFloat(Math.round(dec * 100) / 100).toFixed(2);
-			return "$" + addThousandsSeparators(exchangedAmt);
+			return addThousandsSeparators(exchangedAmt) + " $";
+		}
+		if (exchangeType.toLowerCase() === 'eur' && global.exchangeRates['btceur'] != null && global.exchangeRates['btc'] != null) {
```
Comment (KedziaPawel) [inline app/utils.js:209]: would be nice to move currencies to consts

File: app/utils.js (hunk 1 above)
Comment (mateuszpmroz) [reply app/utils.js:447]: dunno, it's used in other places so I've did it like it was

Comment (KedziaPawel) [reply app/utils.js:447]: looks like a random string :D usually, the first argument is the name of the module so should be logError("utils", {error:error, response:response, body:body}); check if this is the case in logError function

File: app/coins.js [hunk 3]
```diff
@@ -6,5 +6,14 @@ module.exports = {
 	"BTC": btc,
 	"LTC": ltc,
 
-	"coins":["BTCV", "BTC", "LTC"]
-};
\ No newline at end of file
+	"coins":["BTCV", "BTC", "LTC"],
+	"eur": 'eur',
```
Comment (KedziaPawel) [inline app/coins.js:10]: why eur is here and in the currency?

File: views/includes/value-display.pug [hunk 4]
```diff
@@ -1,7 +1,8 @@
 - var currencyFormatInfo = utils.getCurrencyFormatInfo(currencyFormatType);
 
 if (currencyValue > 0)
-	- var parts = utils.formatCurrencyAmount(currencyValue, currencyFormatType).split(" ");
+	- var formattedCurrency = currencyFormatType === "eur" || currencyFormatType === "usd" ? utils.formatExchangedCurrency(currencyValue, currencyFormatType) : utils.formatCurrencyAmount(currencyValue, currencyFormatType);
```
Comment (KedziaPawel) [inline views/includes/value-display.pug:4]: use consts

File: app/coins.js (hunk 3 above)
Comment (mateuszpmroz) [reply app/coins.js:10]: Mistake, shouldn't be here

**Pull request merged**